src/components/Game/Timer/Timer.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import classes from './Timer.scss';
+import { formatDuration } from '../../../helpers';
+import { DURATION, INTERVAL } from '../../../config/config';
+
+
+class Timer extends Component {
+    state = {
+        timeLeft: DURATION,
+    };
```

Comment:
this should be inside a constructor
https://reactjs.org/docs/react-component.html#constructor

Reply:
as far as I know it doesn't have to: https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599

Reply:
I didn't know that, I don't like it, I have never encountered this